prime-angular-frontend/src/app/lib/utils/string-utils.class.ts

```diff
@@ -1,7 +1,8 @@
 export class StringUtils {
   public static capitalize(value: string): string {
+    const strValue: String = new String(value);
```

Comment:
Explicitly wrapping the primitive string into a String object, so that the unit test is happy. Was complaining about charAt() and splice() is undefined function.